listing.php

```diff
@@ -36,7 +37,7 @@
     <?php
         $db = new PDO("mysql:host=db; dbname=listings-feed", "root", "password");
 
-        $agentRef = 'CSL123_100259';
+       $agentRef= $_GET['agentRef'];
```

Comment:
Can't test this right now as there is no agentRef in the get, as I'm not navigating from index.php?